server/routes/index.js

```diff
@@ -0,0 +1,4 @@
+/**
+ * router function loader
+ */
+module.exports = require('../utils/dir-auto-loader')(__dirname);
```

Comment:
__[eslint]__
* Error - 'module' is not defined. (no-undef)
* Error - 'require' is not defined. (no-undef)
* Error - '__dirname' is not defined. (no-undef)